app/components/atomic/galoy-input/galoy-redesigned-input.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react"
+import { TextInput } from "react-native"
```

Comment:
Icon requires margin to the right.

Reply:
added it here https://github.com/GaloyMoney/galoy-mobile/pull/751/commits/ce4235e37f4fe120ca08ca1fddede4c61c19505f#diff-e616ada00da74acb0f4bc85feeb8a6cba93a81aef1c6b2691db1f392a1a8ccd6R45